### ISSUE TITLE
User Request: Nation-wide heat map of biodiversity

### ISSUE THREAD

**Issue by laurabrenskelle** [ioos/marine_life_data_network#83]
### Who is requesting this?

Marine Life community stakeholders broadly

### What is being requested?

"One of the things we are being faced with in America the Beautiful is to conserve 30% of US waters.  We want this conservation to be focused on the most biodiverse areas of the ocean.  However, we do not have a product (ie nation-wide heat map) of where these areas are currently located. There are some data and attempts to do this using trawl surveys but those are focused on higher trophic levels - we are not aware of any product that does this across trophic-levels. We need a map with multiple data layers (phytoplankton diversity, zooplankton diversity, fish diversity, habitat) and perhaps a final layer that is full diversity across trophic levels.  We could then overlay the marine protected areas to see if they match and if there are areas of high biodiversity that we are missing."

This sentiment has been repeated by multiple stakeholders at various meetings so we are capturing this request in this issue.

**Comment by MathewBiddle** [comment]
Related works to address this need:
* obisindicators package - https://github.com/marinebon/obisindicators (see [open issues](https://github.com/marinebon/obisindicators/issues?q=is%3Aopen+is%3Aissue))
  * Initially developed during [2022 Code Sprint](https://ioos.github.io/ioos-code-sprint/2022/topics/04-biodiversity-indicator-development.html). 
* ES50 indicator on a globe - https://mathewbiddle.github.io/globe/ (source: https://github.com/MathewBiddle/globe)

**Comment by laurabrenskelle** [comment]
relates to work here: https://github.com/MathewBiddle/globe

**Comment by laurabrenskelle** [comment]
Maybe this is In Progress now?

**Comment by MathewBiddle** [comment]
Yes! See https://github.com/MathewBiddle/globe/blob/main/GIS4Ocean_h3_indicators.Rmd.

We have a better understanding of what we're doing now. Working with NMS, we are building a GIS layer (.geojson) of various indicators from an OBIS snapshot, using the `obisindicators` R package, to be added to an arcGIS story map. We will use this as an opportunity to highlight what our current capabilities are as well as some of the caveats with building such indicators.

**Comment by MathewBiddle** [comment]
### Who is requesting this? 

NOAA GIS4Ocean @mimidiorio

### What is being requested?
* @MathewBiddle has started a script to generate the geojson files https://github.com/MathewBiddle/globe/blob/main/GIS4Ocean_h3_indicators.Rmd but compute resources are limited on a local laptop.
* Run script on large memory machine to produce resolution 6 indicator H3 hexagons.
* We can pull the OBIS snapshot https://api.obis.org/export

### What is the requested deadline and why? 
2024-10-01
Why? Publish a pilot map layer by end of calendar year.

**Comment by MathewBiddle** [comment]
We will be working in https://github.com/NOAA-GIS4Ocean

**Comment by MathewBiddle** [comment]
After some computational sleuthing 🕵️ I was able to figure out a way to make the higher resolution H3 gridding more performant. You can find the `.Rmd` and resultant `.geojson` files available at https://github.com/NOAA-GIS4Ocean/H3_indicators/tree/main

We will continue working with @mimidiorio to ensure we are generating the appropriate product.

**Comment by MathewBiddle** [comment]
We can generate h3 grids from resolution 1 to 6 and produce [.geojson files](https://github.com/NOAA-GIS4Ocean/H3_indicators/tree/main/data). Below is a screenshot of those files loaded in geojson.io to show the difference in resolutions:

![image](https://github.com/user-attachments/assets/baac026a-fcd3-436b-a8bf-e06d024222b9)

**Comment by MathewBiddle** [comment]
@laurabrenskelle we're at a point where we need to decide what is done for this effort.

We have:

- [x] successfully generated the appropriate indicator heatmaps at the resolution requested. https://github.com/NOAA-GIS4Ocean/H3_indicators/tree/main/data
- [x] created a space to further develop the code, if needed.  https://github.com/NOAA-GIS4Ocean/H3_indicators

So, are we "done" now? Or, are we "done" once the GIS the Ocean storymap has been released?

**Comment by mimidiorio** [comment]
I think we could focus on the write up at this point.  I can pull content from various sources for the indicators, but it would be helpful to have super clear metadata and summary of the strengths and weaknesses of the process.  Candid, honest, what this can and can't do and what the next steps might be to improve indicators moving forward....Thoughts?

**Comment by laurabrenskelle** [comment]
@mimidiorio I think I am ready to start working on this if you would like to tag-up and chat more about it and what format you're looking for this in? Also check out a conversation showing some work we've done with the bio/eco EOVs over here: https://github.com/ioos/ioos_code_lab/pull/220

**Comment by MathewBiddle** [comment]
@laurabrenskelle, @mimidiorio and @MathewBiddle met yesterday to discuss a draft storymap. We highlighted one additional data visualization to work on in addition to providing the text which will accompany the graphics.

* https://github.com/NOAA-GIS4Ocean/H3_indicators/issues/10 

This issue will be closed once the story map is published.

**Comment by MathewBiddle** [comment]
FYI, looking to add the OBIS snapshot parquet file to NODD. We don't need it for this project but it could be beneficial in the future as there continues to be more interest in the data product.

Updates:
* IOOS has drafted text and are in the review phase now.
* Layers have been added to the storymap with draft text.

Actions:
* IOOS to finalize text.

Timeline:
* finalize text for story map: 2025-01-17
* dry launch of story map: 2025-01-31
* ESRI fed launch: 2025-02-28

**Comment by MathewBiddle** [comment]
From my perspective, @laurabrenskelle and @MathewBiddle have provided the text (to the best of our ability). If there are other actions needed for this effort, please request to reopen this ticket.

Otherwise, I'm closing for now.

**Comment by MathewBiddle** [comment]
actually

**Comment by MathewBiddle** [comment]
shoot, getting a little ahead of myself.

I'd like to leave this open until we have a public link to reference to. @mimidiorio please update this issue once a public url is available. Thanks!
